packages/dds/merge-tree/src/test/testClientLogger.ts

```diff
@@ -264,31 +264,37 @@ export class TestClientLogger {
             const node = nodes.shift();
             if (node) {
                 if (node.isLeaf()) {
-                    if (TextSegment.is(node)) {
-                        if (node.parent !== parent) {
-                            if (acked.length > 0) {
-                                acked += " ";
-                                local += " ";
-                            }
-                            parent = node.parent;
+                    if (node.parent !== parent) {
+                        if (acked.length > 0) {
+                            acked += " ";
+                            local += " ";
                         }
+                        parent = node.parent;
+                    }
+                    const text = 
+                        TextSegment.is(node) 
+                            ? node.text 
+                            : Marker.is(node) 
+                                ? "¶"
```

Comment:
this just adds output support for marker nodes. we represent them with the paragraph symbol `¶`